packages/task/src/browser/task-service.ts

```diff
@@ -14,57 +14,57 @@
-import { TerminalService } from '@theia/terminal/lib/browser/base/terminal-service';
-import { TerminalWidget } from '@theia/terminal/lib/browser/base/terminal-widget';
-import { TerminalWidgetFactoryOptions } from '@theia/terminal/lib/browser/terminal-widget-impl';
-import { VariableResolverService } from '@theia/variable-resolver/lib/browser';
-import { WorkspaceService } from '@theia/workspace/lib/browser/workspace-service';
-import { inject, injectable, named, postConstruct } from 'inversify';
-import { DiagnosticSeverity, Range } from 'vscode-languageserver-types';
+import {EditorManager} from '@theia/editor/lib/browser';
+import {ProblemManager} from '@theia/markers/lib/browser/problem/problem-manager';
+import {TerminalService} from '@theia/terminal/lib/browser/base/terminal-service';
+import {TerminalWidget} from '@theia/terminal/lib/browser/base/terminal-widget';
+import {TerminalWidgetFactoryOptions} from '@theia/terminal/lib/browser/terminal-widget-impl';
+import {VariableResolverService} from '@theia/variable-resolver/lib/browser';
+import {WorkspaceService} from '@theia/workspace/lib/browser/workspace-service';
+import {inject, injectable, named, postConstruct} from 'inversify';
+import {DiagnosticSeverity, Range} from 'vscode-languageserver-types';
 import {
+    ApplyToKind,
+    BackgroundTaskEndedEvent,
+    DependsOrder,
     NamedProblemMatcher,
     ProblemMatchData,
     ProblemMatcher,
+    RevealKind,
     RunTaskOption,
     TaskConfiguration,
+    TaskConfigurationScope,
     TaskCustomization,
-    TaskExitedEvent,
-    TaskInfo,
-    TaskOutputProcessedEvent,
-    BackgroundTaskEndedEvent,
     TaskDefinition,
-    TaskServer,
+    TaskExitedEvent,
     TaskIdentifier,
-    DependsOrder,
-    RevealKind,
-    ApplyToKind,
+    TaskInfo,
     TaskOutputPresentation,
-    TaskConfigurationScope
+    TaskOutputProcessedEvent,
+    TaskServer
 } from '../common';
-import { TaskWatcher } from '../common/task-watcher';
-import { ProvidedTaskConfigurations } from './provided-task-configurations';
-import { TaskConfigurationClient, TaskConfigurations } from './task-configurations';
-import { TaskResolverRegistry } from './task-contribution';
-import { TaskDefinitionRegistry } from './task-definition-registry';
-import { TaskNameResolver } from './task-name-resolver';
-import { TaskSourceResolver } from './task-source-resolver';
-import { ProblemMatcherRegistry } from './task-problem-matcher-registry';
-import { TaskSchemaUpdater } from './task-schema-updater';
-import { TaskConfigurationManager } from './task-configuration-manager';
-import { PROBLEMS_WIDGET_ID, ProblemWidget } from '@theia/markers/lib/browser/problem/problem-widget';
-import { TaskNode } from './task-node';
-import { MonacoWorkspace } from '@theia/monaco/lib/browser/monaco-workspace';
-import { TaskTerminalWidgetManager } from './task-terminal-widget-manager';
+import {TaskWatcher} from '../common/task-watcher';
```

Comment:
what happens here? our formatting requires spaces, could you reformat with Theia or VS Code